app/assets/javascripts/app/views/ticket_zoom/article_new.jst.eco

```diff
@@ -39,8 +39,8 @@
     <% end %>
   </div>
   <div class="article-content bubble-gap">
+    <div class="article-visibility-text is-hidden"></div>
```

Comment:
I am not very used to Zammad, could you tell me if there is other places this text should also add the explanation text?